app/routes/events/components/EventEditor/index.js

```diff
@@ -354,7 +349,23 @@ function EventEditor({
                   </Tooltip>
                 </div>
               )}
-
+            {['NORMAL', 'INFINITE'].includes(event.eventStatusType) && (
+              <Tooltip content="Frist for påmelding - antall timer før arrangementet">
```

Comment:
This applies as a hard limit to both "påmelding" and "avmelding/avregistrering".

The other value called "avregistreringsfrist" is a soft limit, when the user will get a penalty for unregistering. Not sure what to say to make the difference clear to the user tho... :thinking:

Reply:
I'm not sure if I follow. 

> hard limit to both "påmelding" and "avmelding/avregistrering"

Does this mean you can't leave an event after this deadline?

Reply:
Yup

Reply:
Maybe add f.ex. "Det er ikke mulig å melde seg av etter denne fristen" in the tooltip to make it a bit clearer?

Reply:
Perhaps we should specify that it applies to both "påmelding" and "avmelding", f. ex. "Det er ikke mulig å melde seg hverken på eller av etter denne fristen"